### PR TITLE
Minor changes to the paper

### LINKED ISSUE #386: Changes to paper
This is part of review over at https://github.com/openjournals/joss-reviews/issues/6934

Small small items from my side. Feel free to choose as needed.

- Shuai Liu is missing ORCid
- Affiliations: some of the have zip codes, some of them dont
- line 23-25 could be better with a general citation
- line 81-84 could be better with a general citation
- line 111 - you could remove the word 'doing'
- line 182 - is it un-equilibrated or non-equilibrated?
- line 198 - formatting for 'alchemlyb' is not there
- line 221 - citation, please check DOI
- line 257 - citation, please recheck DOI
- line 271 - citation, please recheck DOI
- line 293 - citation, please recheck DOI
- some journal names are abbreviations, some are full, could you please make them consistent?

Thanks a lot!

### PR DESCRIPTION
**Pull request by xiki-tempula**
Fix #386
> Shuai Liu is missing ORCid

@shuail 

> Affiliations: some of the have zip codes, some of them dont

All zip code removed

> line 23-25 could be better with a general citation

Add merz2010drug

> line 81-84 could be better with a general citation

Add deng2009computations

> line 111 - you could remove the word 'doing'

Removed

> line 182 - is it un-equilibrated or non-equilibrated?

non-equilibrated

> line 198 - formatting for 'alchemlyb' is not there

Formatted

> line 221 - citation, please check DOI

Added.

> line 257 - citation, please recheck DOI

This is a book and I cannot find a DOI

> line 271 - citation, please recheck DOI

Added

> line 293 - citation, please recheck DOI

Book, cannot find DOI

> some journal names are abbreviations, some are full, could you please make them consistent?

All full now.

### CODE REVIEW

File: joss_paper/paper.bib [hunk 1]
```diff
@@ -254,6 +255,7 @@ @article{scikitlearn2011
   volume={12},
   pages={2825--2830},
   year={2011}
+  doi={10.5555/1953048.2078195}
```
Comment (orbeckst) [inline joss_paper/paper.bib:258]: https://doi.org/10.5555/1953048.2078195 does not resolve so I'd remove it.

**Review by orbeckst — CHANGES_REQUESTED** [review]
THanks for the updates!

@shuail 's ORCID is https://orcid.org/0000-0002-8632-633X (as far as I can tell) but unless he tells us that he wants to use it, we should not add it.

Remove the DOI that doesn't resolve, please.

**Review by orbeckst — APPROVED** [review]

**Comment by orbeckst** [comment]
@xiki-tempula wait with merging. I pinged @shuail via LinkedIn to ask if he wants the ORCID to be added.

**Comment by shuail** [comment]
@orbeckst Hi Oliver, yes, https://orcid.org/0000-0002-8632-633X is mine ORCID, please go ahead and use this in the paper. Thx!

**Comment by orbeckst** [comment]
@xiki-tempula could you please add the missing ORCID and then merge? I’m travelling and don’t have the bandwidth. Thank you!!!

**Pull request merged**